src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -0,0 +1,1024 @@
+
+            ! decomp_k used in SoilBiogeochemPotentialMod.F90
+            ! TODO Defined correctly here?
+            ! TODO Combine multiple terms of same donor pools...
+            ! TODO Where to calculate m1_pool and m2_pool?
+            ! TODO When to introduce litter/soil biomass to the calculation?
+            ! TODO Excluded spinup terms for now. See cwd below how used.
+            decomp_k(c,j,i_met_lit) = (vmax_l1_m1 * m1_pool(c,j) / &
+                                       (km_l1_m1 + m1_pool(c,j))) * &
+                                      w_d_o_scalars
+            decomp_k(c,j,i_met_lit) = (vmax_l1_m2 * m2_pool(c,j) / &
+                                       (km_l1_m2 + m2_pool(c,j))) * &
+                                      w_d_o_scalars
+
+            decomp_k(c,j,i_str_lit) = (vmax_l2_m1 * m1_pool(c,j) / &
+                                       (km_l2_m1 + m1_pool(c,j))) * &
+                                      w_d_o_scalars
+            decomp_k(c,j,i_str_lit) = (vmax_l2_m2 * m2_pool(c,j) / &
+                                       (km_l2_m2 + m2_pool(c,j))) * &
+                                      w_d_o_scalars
+
+            decomp_k(c,j,i_pas_som) = (vmax_s3_m1 * m1_pool(c,j) / &
+                                       (km_s3_m1 + m1_pool(c,j))) * &
+                                      w_d_o_scalars
+            decomp_k(c,j,i_pas_som) = (vmax_s3_m2 * m2_pool(c,j) / &
+                                       (km_s3_m2 + m2_pool(c,j))) * &
+                                      w_d_o_scalars
+
+            decomp_k(c,j,i_act_som) = k_s1_s3 * w_d_o_scalars
+            decomp_k(c,j,i_slo_som) = k_s2_s3 * w_d_o_scalars
+
+            decomp_k(c,j,i_cop_mic) = tau_m1_s1 * m1_pool(c,j)**(densdep) * &
+                                      fphys(c) * w_d_o_scalars
+            decomp_k(c,j,i_cop_mic) = tau_m1_s2 * m1_pool(c,j)**(densdep) * &
+                                      fchem(c) * w_d_o_scalars
+            decomp_k(c,j,i_cop_mic) = tau_m1_s3 * m1_pool(c,j)**(densdep) * &
+                                      faval(c) *w_d_o_scalars
+            decomp_k(c,j,i_oli_mic) = tau_m2_s1 * m2_pool(c,j)**(densdep) * &
+                                      fphys(c) * w_d_o_scalars
+            decomp_k(c,j,i_oli_mic) = tau_m2_s2 * m2_pool(c,j)**(densdep) * &
+                                      fchem(c) * w_d_o_scalars
+            decomp_k(c,j,i_oli_mic) = tau_m2_s3 * m2_pool(c,j)**(densdep) * &
+                                      faval(c) * w_d_o_scalars
+            ! Same for cwd but only if fates not enabled; fates handles cwd on
+            ! its own structure
+            ! TODO Kept cwd bc we do not have a MIMICS replacement for it.
+            !      Also this shows what the BGC decomp_k formulas looked like.
+            if (.not. use_fates) then
+               decomp_k(c,j,i_cwd) = k_frag * w_d_o_scalars * spinup_geogterm_cwd(c)
```

Comment:
should go from cwd to str litter
should be same as bgc where it goes to cel & lig